### PR TITLE
explorer belts can now hold food.

### PR DESCRIPTION
**Pull request by Militaires**
:cl: Basilman
add: Explorer belts can now hold food items.
/:cl:

cause why not.

edit: those edited files dont actually have any changes

### CODE REVIEW

File: code/game/objects/items/weapons/stunbaton.dm [hunk 1]
```diff
@@ -194,4 +194,4 @@
 
 /obj/item/weapon/melee/baton/cattleprod/baton_stun()
 	if(sparkler.activate())
-		..()
+		..()
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/stunbaton.dm]: what

File: code/datums/martial.dm [hunk 2]
```diff
@@ -746,4 +746,3 @@
 	if(wielded)
 		return ..()
 	return 0
-
```
Comment (Fox-McCloud) [inline code/datums/martial.dm]: yeahh.......nu

**Comment by nullbear** [comment]
Just discard the changes in those files. If you're not sure how, open the git shell and type: "git reset --hard ^head" until the files show up in your changes. Discard them, then do "git push -f"

**Comment by Militaires** [comment]
@Fox-McCloud did you even read the fucking PR's description.
you did not, did you.

@nullbear thanks.

**Comment by Jordie0608** [comment]
Those changes are there because you edited the files during the commit history on this branch and git detected there was no newline at the end of them.

**Comment by Fox-McCloud** [comment]
It doesn't matter if I read the description or not, those changes are still in *error* and *wrong*.

Files that you are not altering should not be edited.

**Comment by Militaires** [comment]
@Fox-McCloud 
the point is that these files are not altered jesus christ.

**Comment by optimumtact** [comment]
When will the mining buffs stop?

@KorPhaeron you did this

**Comment by Fox-McCloud** [comment]
But they are in the historical sense--these will be errant commits that will show up in the file history despite the fact nothing has changed on them at all---that's not good

**Comment by Militaires** [comment]
@oranges but it's just a convenience it's not that big of a deal.

plus explorer belts can already held bottles, which means you can have a 100 nutriment bottle with you at all times which is much more efficient that food.

**Comment by Militaires** [comment]
i give up maybe another day

**Pull request closed without merging**